tests/e2e/e2e_test.go

```diff
@@ -44,28 +42,21 @@ func TestMain(m *testing.M) {
 	// Register test flags, then parse flags.
 	framework.RegisterClusterFlags(flag.CommandLine)
 	flag.Parse()
-	klog.SetOutput(ginkgo.GinkgoWriter)
 
 	// check if flags are set and if not cancel the test run
 	if *ImageRepo == "" || *ImageTag == "" || *HelmChart == "" {
-		e2elog.Failf("Required flags not set. Please set -image.repo, -image.tag and -helm-chart")
+		log.Fatal("Required flags not set. Please set -image.repo, -image.tag and -helm-chart")
```

Comment:
Are we ok with the standard logger?

Reply:
at this point the test hasn't started yet, is ok